### ISSUE TITLE
Rule Proposal: Warn if compound class names passed to `by.className` locator

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#58]
Compound classes are not permitted inside the `by.className` locator.

Example of a violation:

```js
element(by.className('md-flex md-body-l flex'));
```

**Comment by alecxe** [comment]
Fixed by 640eeefd68a86e342c0489cf9e1d22623986ce9f.
